lib/main.coffee

```diff
@@ -84,6 +91,13 @@ module.exports =
         @parameters[3] = value
       else @parameters[3] = null
     )
+    @subscriptions.add atom.config.observe('linter-phpcs.showSource', (value) =>
+      if value
+        @parameters[4] = "-s"
+      else @parameters[4] = null
```

Comment:
do you mean remove the `else`?

won't that leave the -s if they change showSource from true to false?